straxen/rucio.py

```diff
@@ -91,32 +101,10 @@ def __repr__(self):
                 representation += f', {attr}: {getattr(self, attr)}'
         return representation
 
-    def _set_remote_imports(self):
-        try:
-            from rucio.client.rseclient import RSEClient
-            from rucio.client.didclient import DIDClient
-            from rucio.common.exception import DataIdentifierNotFound
-            self._did_client = DIDClient()
-            self._id_not_found_error = DataIdentifierNotFound
-            self._rse_client = RSEClient()
-        except (ModuleNotFoundError, RuntimeError) as e:
-            raise ImportError('Cannot work with Rucio remote backend') from e
-
     def find_several(self, keys, **kwargs):
-        if not len(keys):
-            return []
-
-        ret = []
-        for key in keys:
-            if not self._support_superruns(key.run_id):
-                ret.append(False)
-                continue
-            did = key_to_rucio_did(key)
-            if self.did_is_local(did):
-                ret.append(('RucioLocalBackend', did))
-            else:
-                ret.append(False)
-        return ret
+        # for performance, dont do find_several with this plugin
+        # we basically do the same query we would do in the RunDB plugin
+        return np.zeros_like(keys, dtype=bool).tolist()
```

Comment:
You will get here a different behavior whether keys is an iterable or not. 

```
np.zeros_like(1, dtype=bool).tolist()  # return False
np.zeros_like((1, ), dtype=bool).tolist()  # returns [False]
```

Reply:
Find several is always called with an iterable:
https://github.com/AxFoundation/strax/blob/64a371384a14d81038d1b573f9ce99348caa569b/strax/run_selection.py#L35